WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/MigrationHelper.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+struct MigrationHelper {
+    static func isWordPressInstalled() -> Bool {
+        guard let wordPressScheme = URL(string: "wordpress://") else {
```

Comment:
@twstokes There was comment about finding an alternative to `MigrationHelper`. Perhaps we can add a `AppScheme` type with an `UIApplication` extension like this:

```swift
enum AppScheme: String {
    case wordpress = "wordpress://"
}

extension UIApplication {
     func canOpen(app: AppScheme) -> Bool
}
```